mmv1/products/edgecontainer/Cluster.yaml

```diff
@@ -0,0 +1,199 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: "Cluster"
+base_url: "projects/{{project}}/locations/{{location}}/clusters"
+create_url: "projects/{{project}}/locations/{{location}}/clusters?clusterId={{name}}"
+update_verb: :PATCH
```

Comment:
Can you add a handwritten update test so we can test out the update functionality? There are examples of these around that you can copy and replace the relevant configs